blocks/search-results-list-block/features/search-results-list/_children/custom-content.jsx

```diff
@@ -129,91 +74,39 @@ class CustomSearchResultsList extends React.Component {
+			this.state;
+		const { arcSite, promoElements } = this.props;
 
 		return (
-			<div>
-				<div className="search-container">
-					<div>
-						<div className="search-icon-container">
-							<SearchIcon fill="#979797" />
-						</div>
-						<PrimaryFont
-							as="input"
-							type="text"
-							placeholder={this.phrases.t("search-results-blocks.search-input-placeholder")}
-							className="search-bar"
-							onChange={(evt) => this.setState({ value: evt.target.value })}
-						/>
-						<PrimaryFont
-							as="button"
-							type="button"
-							className="btn btn-sm"
-							backgroundColor="primary-color"
-							onClick={() => this.fetchStories(false)}
-						>
-							{this.phrases.t("search-results-block.search-button")}
-						</PrimaryFont>
-					</div>
-					{data && (
-						<p className="search-results-text">
-							{this.phrases.t("search-results-block.search-result-number", {
-								smart_count: totalHits,
-								searchTerm,
-							})}
-						</p>
-					)}
-				</div>
-				<div className="results-list-container">
-					{data &&
-						data.length > 0 &&
-						data.map((element) => {
-							const resizedImageOptions = extractImageFromStory(element)
-								? extractResizedParams(element)
-								: placeholderResizedImageOptions;
-
-							return (
-								<SearchResult
-									key={`result-card-${element._id}`}
-									element={element}
-									arcSite={arcSite}
-									targetFallbackImage={targetFallbackImage}
-									resizedImageOptions={resizedImageOptions}
-									promoElements={promoElements}
-									imageProps={this.imageProps}
-								/>
-							);
-						})}
-					{!!(data && data.length > 0 && data.length < totalHits) && (
-						<div className="see-more">
-							<Button
-								ariaLabel={this.phrases.t("search-results-block.see-more-button-aria-label")}
-								buttonStyle={BUTTON_STYLES.PRIMARY}
-								buttonTypes={BUTTON_TYPES.LABEL_ONLY}
-								text={this.phrases.t("search-results-block.see-more-button")}
-								onClick={() => this.fetchStories(true)}
-							/>
-						</div>
-					)}
-				</div>
-			</div>
+			<Stack className={BLOCK_CLASS_NAME}>
+				<SearchField
+					className={`${BLOCK_CLASS_NAME}__field`}
+					onChange={(event) => this.setState({ value: event.target.value })}
+					onSearch={() => this.fetchStories(false)}
+					searchTerm={searchTerm}
+					showResultsStats={data?.length > 0}
+					totalItems={totalHits}
+				/>
+				<ResultsList
+					arcSite={arcSite}
+					className={`${BLOCK_CLASS_NAME}__results`}
+					content={data}
+					onSearch={() => this.fetchStories(true)}
+					promoElements={promoElements}
+					totalItems={totalHits}
+				/>
+			</Stack>
 		);
 	}
 }
 
 CustomSearchResultsList.propTypes = {
 	customFields: PropTypes.shape({
-		searchContentConfig: PropTypes.contentConfig().tag({
-			group: "Configure Content",
-			label: "Display Content Info",
-		}),
+		// searchContentConfig: PropTypes.contentConfig().tag({
```

Comment:
If we do not need these can we remove?

Reply:
removing proptypes from this since it's not being used in the default at all...